torch/_dynamo/test_minifier_common.py

```diff
@@ -66,6 +66,7 @@ def _run_test_code(self, code):
     def _inject_code(self, patch_code, filename):
         patch_code = f"""\
 {patch_code}
+torch._dynamo.config.fake_tensor_propagation = False
```

Comment:
This seems morally wrong though; you want dynamo to be running as similar to the original configuration as possible, so that you are likely to get the bug. If the bug being minified is related to incorrect striding, turning off fake tensor propagation may cause the bug to go away.